frontend/public/kubevirt/components/utils/lazyRenderer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import * as _ from 'lodash-es';
+import { inject } from '../../../components/utils';
+
+const DEFAULT_THROTTLE_DELAY = 500; // in ms
```

Comment:
I think this is too much. What about 100ms?

Reply:
I tried with 100 and the result was still not perfect. Our component tree is simply too complex and will grow. We can introduce a prop with a default value to enable flexibility in different scenarios.

IMO, page refresh per 0.5sec is still bellow user's recognition. This is the "last mile", there is no automatic processing depending on that. Just the user watching the page for changes. I can play with it, i.e. by trying 250ms as the default - let's see.